runtime-transaction/src/extended_transaction.rs

```diff
@@ -0,0 +1,108 @@
+use {
+    solana_sdk::{
+        hash::Hash,
+        message::{v0::LoadedAddresses, SanitizedMessage},
+        signature::Signature,
+        transaction::{
+            Result, SanitizedTransaction, TransactionAccountLocks, VersionedTransaction,
+        },
+    },
+    std::time::Instant,
+};
+
+/// Sanitized transaction with optional start_time
+#[derive(Debug, Clone, Eq, PartialEq)]
+pub struct ExtendedSanitizedTransaction {
```

Comment:
we're already planning on using `RuntimeTransaction` which is `SanitizedTransaction` plus meta-data. If we're going to include timestamp, we should put it on that type, right?

cc @tao-stones

Reply:
💯